src/otx/algorithms/detection/adapters/mmdet/datasets/tiling.py

```diff
@@ -502,11 +506,24 @@ def merge_maps(self, saliency_maps: Union[List[List[np.ndarray]], List[np.ndarra
         Returns:
             merged_maps (List[list | np.ndarray | None]): Merged saliency maps for each image.
         """
+
+        dtype = None
+        for map in saliency_maps:
+            for cl_map in map:
+                # find first class map which is not None
+                if cl_map is not None and dtype is None:
+                    dtype = map[0].dtype
+                    feat_h, feat_w = map[0].shape
+                    break
+            if dtype is not None:
+                break
+        else:
+            # if None for each class for each image
+            return saliency_maps[: self.num_images]
```

Comment:
I think indent level of else is weird, could you check this?

Reply:
This is for else statement. `else` is executed only if for loop isn't done by break